### ISSUE TITLE
Bound IIFE causes `missing_fat_arrows` to crash

### ISSUE THREAD

**Issue by aseemk** [clutchski/coffeelint#457]
```coffee
do (a=b) =>
    @foo a
```

(You don't even need the second line; it's just for sake of example.)

```
$ coffeelint /path/to/that/file.coffee

/path/to/coffeelint/lib/coffeelint.js:1840
    return ((ref = node.variable) != null ? ref.base.value : void 0) === 'cons
                                                    ^
TypeError: Cannot read property 'value' of undefined
  at MissingFatArrows.module.exports.MissingFatArrows.isConstructor (/path/to/coffeelint/lib/coffeelint.js:1840:53)
  at MissingFatArrows.module.exports.MissingFatArrows.lintNode (/path/to/coffeelint/lib/coffeelint.js:1789:14)
  at /path/to/coffeelint/lib/coffeelint.js:1800:22
  at Block.exports.Base.Base.eachChild (/path/to/coffeelint/node_modules/coffee-script/lib/coffee-script/nodes.js:199:17)
  at MissingFatArrows.module.exports.MissingFatArrows.lintNode (/path/to/coffeelint/lib/coffeelint.js:1798:17)
  at MissingFatArrows.module.exports.MissingFatArrows.lintAST (/path/to/coffeelint/lib/coffeelint.js:1779:10)
  at ASTLinter.module.exports.ASTLinter.lint (/path/to/coffeelint/lib/coffeelint.js:515:43)
  at Object.coffeelint.lint (/path/to/coffeelint/lib/coffeelint.js:307:67)
  at ErrorReport.module.exports.ErrorReport.lint (/path/to/coffeelint/lib/coffeelint.js:660:51)
  at lintFiles (/path/to/coffeelint/lib/commandline.js:70:19)
  at Object.<anonymous> (/path/to/coffeelint/lib/commandline.js:213:21)
  at Object.<anonymous> (/path/to/coffeelint/lib/commandline.js:218:4)
  at Module._compile (module.js:456:26)
  at Object.Module._extensions..js (module.js:474:10)
  at Module.load (module.js:356:32)
  at Function.Module._load (module.js:312:12)
  at Module.require (module.js:364:17)
  at require (module.js:380:17)
  at Object.<anonymous> (/path/to/coffeelint/bin/coffeelint:34:5)
  at Module._compile (module.js:456:26)
  at Object.Module._extensions..js (module.js:474:10)
  at Module.load (module.js:356:32)
  at Function.Module._load (module.js:312:12)
  at Function.Module.runMain (module.js:497:10)
  at startup (node.js:119:16)
  at node.js:906:3
```

Adding `# coffeelint: disable=missing_fat_arrows` doesn't help; the only way to work around this is to `ignore` the rule in `coffeelint.json`.

```
$ coffeelint --version
1.9.6

$ coffee --version
CoffeeScript version 1.9.2

$ node --version
v0.10.33
```

**Comment by AsaAyers** [comment]
Adding `# coffeelint: disable=missing_fat_arrows` still has the rule parse it, it just removes any errors that are produced from the report.

**Comment by AsaAyers** [comment]
This was fixed in 1.9.7

**Comment by aseemk** [comment]
Oh cool, didn't know there was a 1.9.7. Thanks.
